### PR TITLE
Add reserve address for calculating staked ANT

### PR DESCRIPTION
**Pull request by Evalir**

### CODE REVIEW

File: src/web3-contracts.js [hunk 1]
```diff
@@ -353,8 +355,14 @@ export function useAntStaked() {
       Web3EthContract.setProvider('wss://mainnet.eth.aragon.network/ws')
       const ANT_ADDR = getKnownContract('TOKEN_ANT')[0]
       const ant = new Web3EthContract(tokenBalanceOfAbi, ANT_ADDR)
-      const antStaked = await ant.methods.balanceOf(NETWORK_AGENT_ADDR).call()
-      setAntStaked(String(antStaked))
+      const antStakedInAgent = bigNum(
+        await ant.methods.balanceOf(NETWORK_AGENT_ADDR).call()
```
Comment (bpierre) [inline src/web3-contracts.js]: I missed the introduction of `Web3EthContract`, is there a reason why we can’t use `useContract()` (= ethers.js) here?  :thinking:

Comment (sohkai) [reply src/web3-contracts.js]: There isn't; it must've been overlooked when we added in the stats module.

**Comment by sohkai** [comment]
From offline discussion, I don't think this is actually needed since the "reserve address" is actually just the Agent.

File: src/web3-contracts.js (hunk 1 above)
Comment (bpierre) [reply src/web3-contracts.js]: Cool, let’s remove it soon!

I think we could try https://bundlewatch.io/ using [this action](https://github.com/jackyef/bundlewatch-gh-action).

**Pull request merged**